tests/sparseml/transformers/test_transformers.py

```diff
@@ -80,14 +86,33 @@ def _compare_onnx_models(model_1, model_2):
         assert nodes1_count[node] == nodes2_count[node]
 
 
+def _attempt_copy_original_recipe(model_path):
```

Comment:
I understand the purpose of this function, but also would like to stress the "temporariness" of this solution -> should be removed once we move to `recipe.yaml` by default everywhere (if needed) in our framework files.
```suggestion
# TODO: Remove this function once yaml recipes supplied
def _attempt_copy_original_recipe(model_path):
```